Automattic/WP-Job-Manager
# Refactor shortcode class

Breaks up the `WP_Job_Manager_Shortcodes` a bit

Stacked on #2754

### Changes Proposed in this Pull Request

* Move job dashboard shortcode methods to their own class, rename a few of its methods

### Testing Instructions

* `[job_dashboard]` shortcode should keep working





----

| Plugin build for 0177376391d80a0c4078c6a622d4c08f4d09bbeb <a href="#"><img width=600></a> |
| ------------------------------------------------------------ |
| 📦 [Download plugin zip](https://wpjobmanager.com/wp-content/uploads/2024/02/wp-job-manager-zip-2759-01773763.zip)                       |
| ▶️ [Open in playground](https://wpjobmanager.com/playground/?core=2024/02/2759-01773763)             |
